modules/ROOT/pages/database-internals/store-formats.adoc

```diff
@@ -6,10 +6,10 @@ Neo4j's storage engine supports several store formats that describe how data is
 Starting from 5.23, `block` format is the recommended format for Enterprise Edition due to its superior performance and scalability.
 `block` format uses advanced data structures and inlining techniques to enhance data locality, which leads to better resource utilization.
 
-`aligned` is the recommended format for Community Edition.
+`aligned` is the default format for Community Edition starting with Neo4j 5.0.
```

Comment:
In fact, I don't know starting from which version the `aligned` format is the default in CE. It is shot in the dark.
In the section 'Available store formats', down the page, we say that `aligned` is default in CE but don't detail starting from which version.